packages/cli/src/cmds/cmd_build/generate.rs

```diff
@@ -12,5 +15,140 @@ pub fn generate_cxx_bridge_src(rust_crate: RustCrate, module_target_path: impl A
     //
     // TODO: Place generated source in {module}/src/cxxbridge
 
+
+
+    let crate_src = rust_crate.path.join("src");
+    if !crate_src.exists() {
+        cu::debug!("generate_cxx_bridge_src: no src/ at {}", crate_src.display());
+        return Ok(());
+    }
+
+    let module_root = module_target_path.as_ref();
+    let include_rust = module_root.join("include").join("rust");
+    let cxx_src_dir  = module_root.join("src").join("cxxbridge");
+    fs::create_dir_all(&include_rust)
```

Comment:
cu::fs will save the with_context. all trace/debug is built in https://cu.pistonite.dev/pistonite_cu/fs/fn.make_dir.html